test/utils/pickers/describeValue/testPickerOpenCloseLifeCycle.tsx

```diff
@@ -229,6 +229,11 @@ export const testPickerOpenCloseLifeCycle: DescribeValueTestSuite<PickerValidVal
       expect(onAccept.callCount).to.equal(0);
       expect(onClose.callCount).to.equal(0);
 
+      // Close the picker to reset the range position on range pickers
+      fireEvent.keyDown(document.activeElement!, { key: 'Escape' });
+      expect(onAccept.callCount).to.equal(1);
+      expect(onClose.callCount).to.equal(1);
```

Comment:
It seems that we are changing the intent of the test. 🤔 
Maybe we should go to the "next" step if there is a `Next` button? 🤔

Reply:
We do go to `Next` inside `setNewValue`
Before this PR, the picker would close at the end of the selection, the test was able to then re-open the picker and select a new value.
I tried keeping this logic, but now I have to manually go back to the 1st step, and to do that I closed the picker (I don't have a previous step).